File: 2D_sym_python/animals/Sheep.py
from Animal import*

class Sheep(Animal):

    def __init__(self):
        super().__init__()
        self.name='S'
        self.strength = 4
        self.initiative = 4
    
    def GetName(self):
        return 'S'
    
    def GetOrganismFullName(self):
        return "Sheep"

File: 2DVirtualWorldSimulatorCpp/2DVirtualWorldSimulator/Sheep.cpp
#include <iostream>
#include "Sheep.h"

Sheep::Sheep() {
	this->name = 'S';
	this->strength = 4;
	this->initiative = 4;
}
char Sheep::GetName() const {
	return 'S';
}

string Sheep::GetOrganismFullName() const {
	return "Sheep";
}

Sheep::~Sheep() {

}